src/libcore/cell.rs

```diff
@@ -280,6 +281,19 @@ unsafe impl<T> Send for Cell<T> where T: Send {}
 #[stable(feature = "rust1", since = "1.0.0")]
 impl<T> !Sync for Cell<T> {}
 
+#[unstable(feature = "move_cell", issue = "39264")]
+impl<T:Clone> Clone for Cell<T> {
+    #[inline]
+    default fn clone(&self) -> Cell<T> {
+        let temp = unsafe { mem::uninitialized() };
+        let inner = self.replace(temp);
+        // If `clone` panics, let it crash.
```

Comment:
@rkruppe: Thanks for the link! After having a quick read through it and the comments on the original RFC, I think that safely implementing the methods proposed by this PR is fundamentally incompatible with "MoveCells", since we are not allowed to call *any* method on the contained Type, as I understand it. (This is not a problem for "CopyCells" since we can create a new copy of the value without calling any methods on it).